### PR TITLE
docs: create new get started example

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## About this change—what it does

Create a new example for getting started with Aiven Provider for Terraform to replace the outdated example using teams.

### CODE REVIEW

File: examples/get-started/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+```
+
+The output is similar to the following:
+
+```sh
+Terraform v1.6.2
++ provider registry.terraform.io/aiven/aiven v4.9.2
+```
+
+2. Clone this repository.
```
Comment (Serpentiel) [inline examples/get-started/README.md:29]: cloning the repository is only needed for development purposes, for regular usage of the Terraform provider it's not needed

**Review by Serpentiel — CHANGES_REQUESTED** [review]

File: examples/get-started/README.md (hunk 1 above)
Comment (staceysalamon-aiven) [reply examples/get-started/README.md:29]: This step was added here and in other examples in anticipation of creating a dedicated examples repo. That way everything can just be moved without editing the steps - does that work or is it better to still generalize the instructions?

Comment (Serpentiel) [reply examples/get-started/README.md:29]: Ah, I see the point now. We should definitely consider a dedicated examples repo in future. For now, let's get forward with it.

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**